website/src/content/docs/extensions/overview.md

```
@@ -0,0 +1,62 @@
+| [Table](/extensions/table)                         | Editable tables with row, column, and cell selection       |
+| [Text](/extensions/text)                           | The text node every schema requires                        |
+
+## Marks
+
+| Extension                                          | Description                                                |
+| -------------------------------------------------- | ---------------------------------------------------------- |
+| [Background Color](/extensions/background-color)   | Apply a background color to a text selection               |
+| [Bold](/extensions/bold)                           | Bold text formatting backed by the standard `<strong>` mark |
+| [Code](/extensions/code)                           | Inline code formatting backed by the standard `<code>` mark |
+| [Italic](/extensions/italic)                       | Italic text formatting backed by the standard `<em>` mark  |
+| [Link](/extensions/link)                           | Hyperlink mark with optional auto-detection from typed URLs |
```

markdownlint-cli2 (0.22.1)

[warning] Link text should be descriptive

(MD059, descriptive-link-text)